criteria/common/src/org/immutables/criteria/repository/sync/SyncFetcherDelegate.java

```diff
@@ -53,7 +53,7 @@ public T one() {
   @Override
   public Optional<T> oneOrNone() {
     List<T> list = Publishers.blockingListGet(fetcher.oneOrNone());
-    Preconditions.checkState(list.size() <= 1, "Invalid list size: %d", list.size());
```

Comment:
These are actually errorprone errors. It knows that Preconditions doesn't accept anything other than `%s` as a placeholder.